docs/configuration/inter_node_data_path.md

```diff
@@ -137,3 +137,33 @@ paths = arch.search_paths(
     node_filter = node_filter,
     communication_filter = comm_filter)
 ```
+
+### Combine paths
+
+If the path from `source node` to `destination node` is too long, `Architecture.search_paths()` takes a long time and may find many paths that don't need to be considered.
```

Comment:
search_paths searches 15 depths for each relay node.
https://github.com/tier4/CARET_analyze/blob/main/src/caret_analyze/architecture/architecture.py#L244
Even for long paths, the time will not be long if we do the following
```
paths = arch.search_paths('source_node', 'intermediate_node', 'destination_node')[0]
```

```suggestion
```

Reply:
I modified as suggestion.